x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_single_domain.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useParams } from 'react-router-dom';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiCode } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { getEngineBreadcrumbs } from '../engine';
+import { AppSearchPageTemplate } from '../layout';
+
+import { CRAWLER_TITLE } from './constants';
+import { CrawlerSingleDomainLogic } from './crawler_single_domain_logic';
+
+export const CrawlerSingleDomain: React.FC = () => {
+  const { domainId } = useParams() as { domainId: string };
+
+  const { dataLoading, domain } = useValues(CrawlerSingleDomainLogic);
+
+  const { fetchDomainData } = useActions(CrawlerSingleDomainLogic);
+
+  const displayDomainUrl = domain
+    ? domain.url
```

Comment:
Aren't we displaying the domain name only in the standalone UI? Is that something you wanna follow up with or do in this PR?

Reply:
`domain.url` is `domain.name` in the API which I believe is only the domain + protocol, and does not include any entry points